qiskit_optimization/applications/bin_packing.py

```diff
@@ -0,0 +1,97 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2021.
```

Comment:
Minor comment - since this is new code then just the 2021 date is all that is needed.

Reply:
Fix it.